.github/workflows/ci_cd.yml

```diff
@@ -0,0 +1,56 @@
+name: Zulip Desktop CI
+
+on: [push]
+
+jobs:
+  test:
+    name: Test on ${{ matrix.os }}
```

Comment:
You can use `${{ runner.os }}` here to get a readable name of the platform i.e. Linux instead of ubuntu-latest. Also, let's drop the `Test on` part there.

Reply:
Using `${{ runner.os }}` instead of `${{ matrix.os }}` will give error
 `Unrecognized named-value: 'runner'` and the pipeline doesn't run.

Reply:
Hmm, It is documented in the [GitHub Actions docs](https://help.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#runner-context) so I do not know why it won't work? Perhaps, it only work if it is used for steps and env declartions?